scene_select/ard_scene_select.py

```diff
@@ -192,6 +195,16 @@ def process_scene(dataset, days_delta):
     assert dataset.local_path.name.endswith("metadata.yaml")
 
     days_ago = datetime.now(dataset.time.end.tzinfo) - timedelta(days=days_delta)
+    # Continue here if definitive cannot be procduced
+    # since the ancillary files are not there
+    if definitive_ancillary_files(dataset.time.end) is False:
```

Comment:
Should this block have a `return False` at the end?

Reply:
Definitely! Thanks.